src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -64,7 +65,7 @@ public InputEnvelope parseEnvelope(
             schemaValidator.validate(metadataStream, zipFileName);
 
             return schemaValidator.parseMetafile(metadataStream);
-        } catch (JsonParseException exception) {
+        } catch (JsonParseException | JsonMappingException exception) {
```

Comment:
This makes `"additionalProperties": false` feature obsolete or vice versa. Shall we remove extra flag from all occurrences in schema definition?

Reply:
I've made the check stricter, i.e. we're only catching JsonMappingException if it represents OCR data parsing issue. Are you ok with the change?

Reply:
of course ;)